### ISSUE TITLE
Minor Doc fixes

### ISSUE THREAD

**Issue by ankudinov** [aristanetworks/ansible-avd#61]
# To Be Fixed:
## 1.
[Here](https://github.com/aristanetworks/ansible-avd/blob/feature-avd-documentation/ansible_collections/arista/avd/roles/eos_l3ls_evpn/README.md#role-inputs-and-outputs) to be changed from:  
```
As mentioned in the overview, if desired, the role can be extended to leverage data from dynamic sources such as an IPAM or CMDB.
```
to:  
```
If desired, the role can be extended to leverage data from dynamic sources such as an IPAM or CMDB.
```

**Comment by ankudinov** [comment]
## 2.
[This README](https://github.com/aristanetworks/ansible-avd/blob/feature-avd-documentation/ansible_collections/arista/avd/roles/eos_l3ls_evpn/README.md) looks like a clone of the main one. Consider making one of them a reference.

**Comment by carlbuchmann** [comment]
resolved with: 9dd81f6257590afb8bc1e6b54b9e0c331c8e99e3 and 12435857a8ac97f588ee252f97b132db6cf0c879
